### ISSUE TITLE
the batch normalization layer of resnet output NaN

### ISSUE THREAD

**Issue by Ereebay** [pangwong/pytorch-multi-label-classifier#7]
I tried the resnet50 layer as the classification model,  and I  changed the input size of the image, but the loss of validation step was nan. I found the problem was the batch normalization layer, the output of it was NaN. Do you have any idea about it?
